.github/workflows/claude.yml

```
@@ -0,0 +1,50 @@
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+
+      - name: Run Claude Code
+        id: claude
+        uses: anthropics/claude-code-action@v1
+        with:
+          claude_code_oauth_token: ${{ secrets.CLAUDE_CODE_OAUTH_TOKEN }}
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)